browser/js/common/directives/navbar/navbar.js

```diff
@@ -1,47 +1,71 @@
-app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state) {
+app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state, $window) {
 
-    return {
-        restrict: 'E',
-        scope: {},
-        templateUrl: 'js/common/directives/navbar/navbar.html',
+	return {
+		restrict: 'E',
+		scope: {},
+		templateUrl: 'js/common/directives/navbar/navbar.html',
 
-        link: function (scope) {
+		link: function (scope) {
+			var temp = 0;
+			scope.goToSection = function(val, state) {
+				if (val !== 'null') {
```

Comment:
What I was trying to do hear was handle the case when we just want to switch state but not scroll down afterwards and not have an error be thrown. This would be for pages that only have one section like the history page. This was an attempt to replicate overloading functions like in Java since JS doesn't allow overloading natively.